scripts/module/stadium.nut

```diff
@@ -0,0 +1,25 @@
+// Stadium billboards
+
+currentModelOffset <- 0;
+BillboardsBegin <- 2042;
+
+function onScriptLoad() {
+  billboard <- CreateObject(BillboardsBegin, 1, -1037.08, 1340.258, 36.552, 255);
```

Comment:
Avoid code duplication.

Call `SetStadiumBillboard(0)` here, and make sure the function handles `if (!billboard) { billboard.delete() }`